### ISSUE TITLE
Encrypt : Unknown status message: 'FAILURE' with gnupg version > 2.1.7-1

### ISSUE THREAD

**Issue by Jonathanm10** [isislovecruft/python-gnupg#143]
OS: Linux jonathan 4.4.3-1-ARCH #1 SMP PREEMPT Fri Feb 26 15:09:29 CET 2016 x86_64 GNU/Linux

python-gnupg: 2.0.2
gnupg : > 2.1.7-1

``` python
Python 3.5.1 (default, Mar  3 2016, 09:29:07) 
[GCC 5.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import gnupg
>>> gnupg.__version__
'2.0.2'
>>> gpg = gnupg.GPG()
>>> encrypt = gpg.encrypt('foo', 'bar')
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/site-packages/gnupg/_meta.py", line 626, in _read_response
    result._handle_status(keyword, value)
  File "/usr/lib/python3.5/site-packages/gnupg/_parsers.py", line 1506, in _handle_status
    super(Crypt, self)._handle_status(key, value)
  File "/usr/lib/python3.5/site-packages/gnupg/_parsers.py", line 1419, in _handle_status
    raise ValueError("Unknown status message: %r" % key)
ValueError: Unknown status message: 'FAILURE'
```

Since newer version of gnupg, python-gnupg cannot handle new error when encrypting with emails not in key ring. And that exception cannot be catch from the code.

Thanks in advance taking care of it

Cheers,

Jonathan

**Comment by isislovecruft** [comment]
Related to #172, and with probably the same solution.

**Comment by isislovecruft** [comment]
@Jonathanm10 Thanks for reporting this!

**Comment by isislovecruft** [comment]
@keisetsu (re: https://github.com/SexualHealthInnovations/callisto-core/issues/115) I didn't realise Project Callisto uses my code… I'm a huge fan of your work! I don't have much time for maintaining this library these days, but if there's ever a bug that's harming Callisto users, I'd be happy to prioritise getting it fixed. Please feel free to email me at isis@torproject.org.

**Comment by keisetsu** [comment]
@kelseyq and other have done the work, I just put in a couple of PRs. Thought she'd like to be in on this.

**Comment by isislovecruft** [comment]
This should be fixed in version 2.2.0 which I'll release tonight after I fix a couple more bugs related to using gpg-2.1.x.

**Comment by kelseyq** [comment]
wowowow this is so cool to see @isislovecruft! the sentiment is *definitely* returned. we've had a easy path with this library (aside from local development hiccups) so far but I will take you up on that offer if I need to for sure.
